flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -119,18 +119,6 @@
 					text("'region': Restarts all tasks that could be affected by the task failure.")
 				).build());
 
-	/**
-	 * This option specifies the interval in order to trigger a resource manager reconnection if the connection
-	 * to the resource manager has been lost.
-	 *
-	 * <p>This option is only intended for internal use.
-	 */
-	public static final ConfigOption<Long> RESOURCE_MANAGER_RECONNECT_INTERVAL =
-		key("jobmanager.resourcemanager.reconnect-interval")
-		.defaultValue(2000L)
-		.withDescription("This option specifies the interval in order to trigger a resource manager reconnection if the connection" +
-			" to the resource manager has been lost. This option is only intended for internal use.");
-
```

Comment:
This requires an update of the docs via `mvn package -Dgenerate-config-docs -pl flink-docs`